libs/cde-visualization/src/lib/cde-visualization/cde-visualization.component.ts

```diff
@@ -1,12 +1,182 @@
 import { CommonModule } from '@angular/common';
+import { VisualizationHeaderComponent } from '../components/visualization-header/visualization-header.component';
+import { Metadata, MetadataComponent } from '../components/metadata/metadata.component';
+import { CellTypeOption, CellTypesComponent } from '../components/cell-types/cell-types.component';
+
+export interface Node {
+  x: number;
+  y: number;
+  z?: number;
+  cell_type: string;
+  cell_ontology_id?: string;
+}
+
+export interface ColorMapItem {
+  cell_type: string;
+  cell_color: string;
+}
+
+const DEFAULT_CELL_TYPE_ANCHOR = 'Endothelial';
+const DEFAULT_CELL_TYPE_COLOR = '#5D667F';
 
 @Component({
-  selector: 'cde-cde-visualization',
+  selector: 'cde-visualization',
   standalone: true,
-  imports: [CommonModule],
+  imports: [CommonModule, VisualizationHeaderComponent, MetadataComponent, CellTypesComponent],
   templateUrl: './cde-visualization.component.html',
   styleUrl: './cde-visualization.component.scss',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class CdeVisualizationComponent {}
+export class CdeVisualizationComponent {
+  readonly nodes = input<Node[]>([
+    {
+      x: 3450.144,
+      y: 2933.142,
+      cell_type: 'NK',
+    },
+    {
+      x: 4492.808,
+      y: 3564.456,
+      cell_type: 'NK',
+    },
+    {
+      x: 5256.62,
+      y: 2724.436,
+      cell_type: 'NK',
+    },
+    {
+      x: 6570.342,
+      y: 2044.626,
+      cell_type: 'NK',
+    },
+    {
+      x: 2688.064,
+      y: 673.748,
+      cell_type: 'MUC1+ Enterocyte',
+    },
+    {
+      x: 3450.144,
+      y: 2933.142,
+      cell_type: 'T Cell',
+    },
+    {
+      x: 4492.808,
+      y: 3564.456,
+      cell_type: 'B Cell',
+    },
+    {
+      x: 5256.62,
+      y: 2724.436,
+      cell_type: 'Macrophage',
+    },
+    {
+      x: 6570.342,
+      y: 2044.626,
+      cell_type: 'Dendritic Cell',
+    },
+    {
+      x: 2688.064,
+      y: 673.748,
+      cell_type: 'Neutrophil',
+    },
+    {
+      x: 7892.234,
+      y: 4567.891,
+      cell_type: 'Epithelial Cell',
+    },
+    {
+      x: 2345.678,
+      y: 9012.345,
+      cell_type: 'Endothelial Cell',
+    },
+    {
+      x: 6789.012,
+      y: 3456.789,
+      cell_type: 'Fibroblast',
+    },
+    {
+      x: 1234.567,
+      y: 8901.234,
+      cell_type: 'Adipocyte',
+    },
+    {
+      x: 5678.901,
+      y: 2345.678,
+      cell_type: 'Astrocyte',
+    },
+    {
+      x: 9012.345,
+      y: 6789.012,
+      cell_type: 'Microglia',
+    },
+    {
+      x: 3456.789,
+      y: 1234.567,
+      cell_type: 'Oligodendrocyte',
+    },
+    {
+      x: 8901.234,
+      y: 5678.901,
+      cell_type: 'Pericyte',
+    },
+    {
+      x: 2345.678,
+      y: 9012.345,
+      cell_type: 'Chondrocyte',
+    },
+    {
+      x: 6789.012,
+      y: 3456.789,
+      cell_type: 'Osteocyte',
+    },
+  ]);
+
+  readonly cellTypeAnchor = input<string>();
+
+  readonly colorMap = input<ColorMapItem[]>();
```

Comment:
Like the `nodes` input this need to accept a url as well

Reply:
now accepts url